docker/Dockerfile

```diff
@@ -0,0 +1,62 @@
+FROM ubuntu:20.04 as web-base
+
+RUN apt update && apt install -y git curl
+RUN curl -sL https://deb.nodesource.com/setup_14.x -o nodesource_setup.sh && \
+    bash ./nodesource_setup.sh && \
+    apt install nodejs && \
+    rm ./nodesource_setup.sh
+
+FROM web-base as web-sdk-builder
+RUN apt update
```

Comment:
Is this 'update' necessary since it was done in the base image?

Reply:
yeah... drop it.